index.html

```diff
@@ -1557,6 +1557,27 @@ <h2 id="sgHeader" class="accordion-header">
                     </div>
                 </form>
             </div>
+            <div class="modal" tabindex="-1" id="confirmDeleteRoundModal">
+                <div class="modal-dialog">
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <h5 class="modal-title">Delete Round?</h5>
+                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                        </div>
+                        <div class="modal-body">
+                            <p>Do you really want to delete that round?</p>
+                        </div>
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">
+                                No, Cancel
+                            </button>
+                            <button type="button" id="confirmDeleteBtn" class="btn btn-primary" onclick="">
```

Comment:
Try to leave JS out of your HTML. You are also already using an event handler to get this button in the confirmDelete function in the roundsMode.js